src/db/query.js

```diff
@@ -1,4 +1,5 @@
-import { fireEvent } from '../event'
+import {fireEvent} from '../event'
+import {PLATFORM} from "../platform";
```

Comment:
Can we change this to `import {PLATFORM} from '../platform'`, sorry didn't catch it the first time.

Reply:
Another one we need to add to our linter